src/main/webapp/app/service/firebase/firebase-gene-review-service.spec.ts

```diff
@@ -38,9 +38,9 @@ describe('Firebase Gene Review Service', () => {
     jest.useFakeTimers().setSystemTime(DEFAULT_DATE);
   };
 
-  describe('updateReviewableContent', () => {
-    beforeEach(() => _beforeEach());
+  beforeEach(_beforeEach);
```

Comment:
Is there a need to have the _beforeEach function as a variable if it's only being used once?

Reply:
I agree that we can get rid of the function